flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/JsonExistsFunctionITCase.java

```diff
@@ -47,6 +47,17 @@
         }
 
         final String jsonValue = IOUtils.toString(jsonResource, Charset.defaultCharset());
+        if (1 + 1 > 0) {
```

Comment:
typo ;-)

Reply:
Typo is a nice word for accidentally comitting this non-sense… :-)